test/helpers/system-test-configuration.js

```diff
@@ -22,14 +22,15 @@ if (process.env.GOSHAWKDB_DEFAULT_CLIENT_KEYPAIR) {
 	clientKeyPath = process.env.GOSHAWKDB_DEFAULT_CLIENT_KEYPAIR
 }
 
-let clusterHosts = process.env.GOSHAWKDB_DEFAULT_CLUSTER_HOSTS || "localhost:7894;"
-const firstHost = clusterHosts.substring(0, clusterHosts.indexOf(':'))
+const clusterHosts = process.env.GOSHAWKDB_DEFAULT_CLUSTER_HOSTS || "localhost:7895;"
```

Comment:
Eeek, this is a bit dodge. For the other clients, this will be set to the capnp/tcp port, and I would prefer not having to change env vars for different clients. So, I wonder if it's best to introduce a separate env var for the wss?
`GOSHAWKDB_DEFAULT_CLUSTER_HOSTS_WSS` ?